operator/app/src/main/java/org/keycloak/operator/v2alpha1/KeycloakDeployment.java

```diff
@@ -403,6 +402,60 @@ private void configureTLS(Deployment deployment) {
             deployment.getSpec().getTemplate().getSpec().getVolumes().add(volume);
             kcContainer.getVolumeMounts().add(volumeMount);
         }
+
+        var userRelativePath = readConfigurationValue(Constants.KEYCLOAK_HTTP_RELATIVE_PATH_KEY);
+        var kcRelativePath = (userRelativePath == null) ? "/" : userRelativePath;
+        var protocol = (this.keycloakCR.getSpec().isHttp()) ? "http" : "https";
+        var kcPort = (this.keycloakCR.getSpec().isHttp()) ? Constants.KEYCLOAK_HTTP_PORT : Constants.KEYCLOAK_HTTPS_PORT;
+
+        var baseProbe = new ArrayList(List.of("curl", "--head", "--fail", "--silent"));
```

Comment:
Just a nitpick to avoid compiler (and IDE) warnings:
```suggestion
        var baseProbe = new ArrayList<>(List.of("curl", "--head", "--fail", "--silent"));
```

Reply:
done